### PR TITLE
Release 0.29.0

### PR DESCRIPTION
**Pull request by pokey**
## Checklist

- [-] I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/)
- [x] I have updated the [docs](https://github.com/cursorless-dev/cursorless/tree/main/docs) and [cheatsheet](https://github.com/cursorless-dev/cursorless/tree/main/cursorless-talon/src/cheatsheet)
- [-] I have not broken the cheatsheet

### CODE REVIEW

File: packages/cursorless-org-docs/src/docs/user/release-notes/0.29.0.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+We also have an internal experiment with a version of Cursorless that runs literally anywhere, using accessibility APIs. This one is so experimental that I'll probably get in trouble for even mentioning it. So definitely don't heckle @AndreasArvidsson and @phillco to tell you how to play with this completely not-ready-for-primetime early experiment.
+
+### New hat shapes
+
+It's unfortunate that we don't record our Discord sessions, because the discussions about the new hat shapes would make fantastic examples for the Wikipedia page on [bike-shedding](https://en.wikipedia.org/wiki/Law_of_triviality). I'm really happy with how they turned out. Except for fox. Such a shame about that pixel.
+
+### Strange Loop 2023
+
+ICYMI Cursorless went to Strange Loop. The talk was so wild that they had to close Strange Loop forever. Sorry. I really liked Strange Loop too. Here's the talk ([video 🎬](https://youtu.be/NcUJnmBqHTY)); I think it's a nice way to understand the concepts underlying Cursorless. Its Platonic form, if you will. And I really hope that you will.
```
Comment (phillco) [inline packages/cursorless-org-docs/src/docs/user/release-notes/0.29.0.md:46]: Worth it

Comment (phillco) [inline packages/cursorless-org-docs/src/docs/user/release-notes/0.29.0.md:46]:
```suggestion
ICYMI Cursorless went to Strange Loop. The talk was so wild that they had to close Strange Loop forever. Sorry. I really liked Strange Loop too. Here's the talk ([video 🎬](https://youtu.be/NcUJnmBqHTY)); I think it's a nice way to understand the concepts underlying Cursorless -- its platonic form, if you will. (And I really hope that you will.)
```

File: packages/cursorless-org-docs/src/docs/user/release-notes/0.29.0.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+We also have an undocumented, secret action that has a built-in parser for building arbitrarily powerful custom commands on the fly. You should also definitely not heckle @AndreasArvidsson and @phillco to tell you how to use this one either.
+
+### Experimental keyboard interface
+
+The keyboard interface continues to grow and is getting close to feature-parity with our spoken grammar, but is still very much in the experimental phase. We're looking for feedback on this one, so if you're interested in trying it out, come find us on Slack or Discord.
+
+### Falling back
+
+You know when you're in a text box or the VSCode integrated terminal and you're like "Man, I wish I could bring that line right to my cursor"? Do it. Go on. `"bring line air"`. You can also use some hatless Cursorless commands on your cursor in that text box (`"take token"`, `"copy two tokens backward"`, etc etc etc 🙄)
```
Comment (phillco) [inline packages/cursorless-org-docs/src/docs/user/release-notes/0.29.0.md:60]: Why the eye roll?

**Review by phillco — APPROVED** [review]

File: packages/cursorless-org-docs/src/docs/user/release-notes/0.29.0.md (hunk 2 above)
Comment (pokey) [reply packages/cursorless-org-docs/src/docs/user/release-notes/0.29.0.md:60]: Just showing off about how many different things you can do. Kind of like "yeah whatever no big deal"

File: packages/cursorless-org-docs/src/docs/user/release-notes/0.29.0.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
```
Comment (pokey) [inline packages/cursorless-org-docs/src/docs/user/release-notes/0.29.0.md:60]: Better?

```suggestion
You know when you're in a text box or the VSCode integrated terminal and you're like "Man, I wish I could bring that line right to my cursor"? Do it. Go on. `"bring line air"`. You can also use some hatless Cursorless commands on your cursor in that text box (`"take token"`, `"copy two tokens backward"`, etc etc etc nbd 🙄)
```

File: packages/cursorless-org-docs/src/docs/user/release-notes/0.29.0.md (hunk 2 above)
Comment (phillco) [reply packages/cursorless-org-docs/src/docs/user/release-notes/0.29.0.md:60]: Ah, I think 😅 or 😎 would capture that intent better

File: packages/cursorless-org-docs/src/docs/user/release-notes/0.29.0.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+We also have an undocumented, secret action that has a built-in parser for building arbitrarily powerful custom commands on the fly. You should also definitely not heckle @AndreasArvidsson and @phillco to tell you how to use this one either.
+
+### Experimental keyboard interface
+
+The keyboard interface continues to grow and is getting close to feature-parity with our spoken grammar, but is still very much in the experimental phase. We're looking for feedback on this one, so if you're interested in trying it out, come find us on Slack or Discord.
+
+### Falling back
+
+You know when you're in a text box or the VSCode integrated terminal and you're like "Man, I wish I could bring that line right to my cursor"? Do it. Go on. `"bring line air"`. You can also use some hatless Cursorless commands on your cursor in that text box (`"quad wrap token"`, `"copy two tokens backward"`, `"chuck third previous paint"`, etc etc etc nbd 🙄)
```
Comment (AndreasArvidsson) [inline packages/cursorless-org-docs/src/docs/user/release-notes/0.29.0.md:60]: I don't think two tokens or ordinals work in our fallback code.

**Pull request merged**